### PR TITLE
Add Dayou Zhou bio

### PR DESCRIPTION
**Pull request by dyzhou222**

### CODE REVIEW

File: bios/dyzhou222.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+**Name:** Dayou Zhou
+
+**Occupation:** Software Engineer
```
Comment (OluOyedipe) [inline bios/dyzhou222.md:3]: In line reviews are pretty cool.

**Comment by dyzhou222** [comment]
Looks good :+1:

File: bios/dyzhou222.md (hunk 1 above)
Comment (dyzhou222) [inline bios/dyzhou222.md:3]: @OluOyedipe so a pull request is like a DR for us??

Comment (OluOyedipe) [inline bios/dyzhou222.md:3]: Yes. Though i think Blaise wants to use JIRA.

**Pull request merged**